dataplane/pkg/common/egress_interface.go

```diff
@@ -105,6 +150,11 @@ func NewEgressInterface(srcIp net.IP) (EgressInterface, error) {
 		return nil, err
 	}
 
+	arpEntries, err := getArpEntries()
+	if err != nil {
```

Comment:
@artem-belov we should check if there is no file, we should not fail.

Reply:
Done